src/tests/infos_docs.edn

```diff
@@ -119,8 +119,10 @@
    ;; details (optional)
    (When
     (-> name (getFilename (str basePath detailsPath) ".md") (FS.IsFile))
-    (-> "--8<-- \"" >> .o name (getFilename detailsPath ".md") >> .o "\"\r\n" >> .o
-        "\r\n\r\n" >> .o))
+    (-> 
+     "## Details\r\n\r\n" >> .o
```

Comment:
Not a rule, but something I have imposed myself for consistency, although I might have broken it sometimes 😉.

I usually don't go to the next line with the `(->)` block. I find it easier to read. Contrary to other blocks that have a name/word describing them, I consider this one to be "structural".